Testscripts/Linux/utils.sh

```diff
@@ -2745,6 +2745,8 @@ function install_lagscope () {
 
 # Build and install ntttcp
 function build_ntttcp () {
+	# Clean up ntttcp before install
+	rm -rf /usr/local/bin/ntttcp
```

Comment:
Can we update ntttcp to use same version for udp and tcp?

Reply:
Does the ntttcp of master branch support both udp and tcp now? I mean to update ntttcp, not in lisa.

Reply:
Yes, master branch can support udp and tcp.